src/main/app/package.json

```diff
@@ -32,6 +32,8 @@
     "@ngx-translate/http-loader": "^8.0.0",
     "@tanstack/angular-query-experimental": "^5.53.2",
     "@types/proj4": "^2.5.4",
+    "@formio/angular": "^7.0.0",
+    "bootstrap": "^4.6.2",
```

Comment:
Bit of a same as you also mentioned that we now need to include bootstrap. I'm guessing this is making our frontend JS quite a lot bigger as well or not? Does this mean all our frontend pages will get a lot bigger in that senses or is that not the case? cc @infonl-marcel

Reply:
The current bundle is exceeding already the threshold by 50% (warning when building). Discussed that with Felix, but he seemed quite relaxed. Gemeente has enough bandwidth.

For common public web apps the bundle size is way too large atm; we can discuss improving it. But those two new packages will not harm compared to current size.

see https://bundlephobia.com/package/@formio/angular@7.0.0 - and Bootstrap is even smaller.

A future improvement could be lazy loading the whole ZAPS part of the for example. Then these JS/TS packages only get loaded when a functioneel beheerder opens the ZAPS page. And then even more granular lazy loading could be considered when opening pages from the ZAPS menu.

Summary: no (extra) worries for going on holidays @edgarvonk  😉

Reply:
thanks @infonl-marcel !